### DISCUSSION TITLE
Find a copy of paper

### DISCUSSION THREAD

**Discussion by VoidScute** [sarnold/cccc#29] [General]
Just to know if someone have or can find the thesis paper, the link from sourceforge are broken.
Thx for keep the tool working.

**Comment by tim-littlefair** [comment]
There is a copy of my PhD thesis shared via Google Drive at [https://docs.google.com/file/d/0B8R9PxuHI6kYQkdlQ0U1SXpQOXc/edit](https://docs.google.com/file/d/0B8R9PxuHI6kYQkdlQ0U1SXpQOXc/edit)
